.codecov.yml

```diff
@@ -0,0 +1,13 @@
+coverage:
+  precision: 1      # Report coverage to 1 decimal place
+  round: down       # Round down
+  range: "90...100" # Anything below 90% is coloured red (considered bad)
+  status:           # Various customisations that it seems should be off
+    project: off
+    patch: off
+    changes: off
+
+comment: false      # Do not post a comment on PRs with a code coverage report
```

Comment:
I'm basically happy with this PR, I wonder why we don't want comments on PR's though? This is far superior to the rudimentary coverage checks we did previously.

Reply:
Yeah, let's do it and see how we find it. Was just wanting to get something minimal started with this PR. We might want to tweak exactly what the comments look like, what information they contain, but we can tweak that later.